ModPatches/Spacer Arsenal/Patches/Spacer Arsenal/Apparel_Various.xml

```diff
@@ -0,0 +1,97 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Patch>
+
+	<!-- Body Armour -->
+
+	<Operation Class="PatchOperationAdd">
+		<xpath>Defs/ThingDef[defName="DV_Apparel_EnsignArmor"]</xpath>
+		<value>
+			<statBases>
+				<Bulk>80</Bulk>
+				<WornBulk>12</WornBulk>
+				<ReloadSpeed>0.1</ReloadSpeed>
+				<Mass>30</Mass>
+				<ArmorRating_Sharp>16</ArmorRating_Sharp>
+				<ArmorRating_Blunt>38.25</ArmorRating_Blunt>
+				<MaxHitPoints>400</MaxHitPoints>
+			</statBases>
+		</value>
+	</Operation>
+
+	<!-- Increase Devilstrand Cost due to much higher heat protection -->
+	<Operation Class="PatchOperationAdd">
+		<xpath>Defs/ThingDef[defName="DV_Apparel_EnsignArmor"]</xpath>
+		<value>
+			<costList>
+				<DevilstrandCloth>60</DevilstrandCloth>
+				<Plasteel>140</Plasteel>
```

Comment:
Just to confirm, is the small increase in Plasteel cost and decrease in Uranium cost intended?

Reply:
Yes when patching I noticed all the other power armour bodies used rounded values to the nearest 10 for all but components and helmets had increments of 5 used. I assumed there was some balance or other reason for this so I tweaked the values very slightly to align with the other armours. 

I upped the plasteel cost as the armour comes with some powerful buffs in the form of a 20% cooldown reduction so believed that armour needed a slightly higher cost to it and plasteel is more expensive and as such harder to acquire than uranium.